src/modules/user/components/professional-profile-tabs/professional-profile-tabs.tsx

```diff
@@ -34,6 +35,8 @@ export const ProfessionalProfileTabs: FC = () => {
   const tabsRef = useRef<HTMLDivElement | null>(null);
   const skipNextScroll = useRef<boolean>(false);
   // observers
+  // Todo:
+  // @ts-ignore
   const entry = useIntersectionObserver(tabsRef, {});
```

Comment:
We need to update usage of this hook according to this doc - https://usehooks-ts.com/react-hook/use-intersection-observer

```
const { ref, isIntersecting } = useIntersectionObserver({
  threshold: 0.5
});

<SomeElement ref={ref} .... />
```

By the way. As I remember we don't use this component `professional-profile-tabs.tsx`. So please, double check it and remove. Also, search for other places where `useIntersectionObserver` is used (there is a few).